addons/test_mail_full/tests/test_sms_performance.py

```diff
@@ -109,12 +109,11 @@ def setUp(self):
     @mute_logger('odoo.addons.sms.models.sms_sms')
     @users('employee')
     @warmup
-    def test_composer_mass_active_domain(self):
+    def test_sms_composer_mass(self):
         composer = self.env['sms.composer'].with_context(
             default_composition_mode='mass',
             default_res_model='mail.test.sms',
-            default_use_active_domain=True,
-            active_domain=[('id', 'in', self.records.ids)],
+            active_ids=self.records.ids,
```

Comment:
Would be interesting to see if that has any impact on the query count here below, maybe we could lower it?